src/style.css

```diff
@@ -1 +1,90 @@
-/* add styles here */
+:root {
+  --font-color: #3d4e61;
+  --linear-gradient-focus: linear-gradient(180deg, #fff, #f6f6f7);
+  --border-input: 1px solid #e1e7ed;
+  --shadow-input: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  --shadow-input-hovered: 0 3px 3px 0 rgba(61, 78, 97, 0.1);
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+.search-bar {
+  margin-top: 20px;
+}
+
+.search-bar__input {
+  box-sizing: border-box;
+  width: 100%;
+  border-radius: 4px;
+  border: var(--border-input);
+  box-shadow: var(--shadow-input);
+  font-family: inherit;
+  color: var(--font-color);
+}
+
+.big {
+  height: 70px;
+  padding-left: 62px;
+  background: url(images/Search.svg) 26px 50% no-repeat;
+  font-size: 16px;
+}
+
+.small {
```

Comment:
you can create one base class, which will feature all styles for bigger input 'search-bar__input' for example, and then you can create 'search-bar__input--small` where you will override needed styles for smaller input